src/content_mapblock.cpp

```diff
@@ -526,15 +543,16 @@ void mapblock_mesh_generate_special(MeshMakeData *data,
 			{
 				v3s16 dir = side_dirs[i];
 
+				NeighbourData& neighbour_data = neighbour_datas[NeighbourToIndex(dir)];
 				/*
 					If our topside is liquid and neighbor's topside
 					is liquid, don't draw side face
 				*/
 				if(top_is_same_liquid &&
-						neighbor_flags[dir] & neighborflag_top_is_same_liquid)
+					neighbour_data.flags & neighborflag_top_is_same_liquid)
```

Comment:
He means that if the if statement logic goes onto the next line, it needs an extra indentation space, so you can easily disambiguate a continuation of the if statement logic from the if statement body.